src/components/molecules/Visualizer/Engine/Cesium/index.tsx

```diff
@@ -74,6 +83,7 @@ const Cesium: React.ForwardRefRenderFunction<EngineRef, EngineProps> = (
           width: small ? "300px" : "auto",
           height: small ? "300px" : "100%",
           display: ready ? undefined : "none",
+          cursor: isLayerDragging ? "grab" : "default",
```

Comment:
Before adding this here, did you try to add the cursor effects on the layer itself? I feel like this is a little hacky and it would be amazing if the cursor could show feedback on hover of each layer (then the user will know somehting is possible in the correct place). 
Currently if I hover over a layer it stays the default, which makes it hard to click correctly on the layer.

Reply:
If we want to add the cursor effects on the layer itself, I assume we need to force `primitive` to have `onHover` props. We can't guarantee that all primitives will have that since they might be developed by external developers.